rollup.config.mjs

```diff
@@ -0,0 +1,24 @@
+import commonjs from '@rollup/plugin-commonjs';
+import typescript from '@rollup/plugin-typescript';
+import postcss from 'rollup-plugin-postcss';
+
+export default {
+  input: 'src/index.ts',
+  output: {
+    dir: 'lib',
+    format: 'es',
+    preserveModules: true,
+    preserveModulesRoot: 'src',
+  },
+  external: ['../../icons/spritemap/spritemap.svg', 'react-children-by-type'],
+  plugins: [
+    commonjs(),
```

Comment:
Convert some CJS modules into ESM. Needed so that rollup can handle some of the node_modules deps.

I think an alternative would be to compile EDS to CJS instead of ESM (wow, those acronyms).